docs/docs/components/calendar-input.md

```diff
@@ -0,0 +1,126 @@
+                <CalendarDemo
+                        label="Pick a Gregorian date"
+                        component={CalendarInput}
+                        calendar="gregory"
+                />
+        </div>
+        <div>
+                <CalendarDemo
+                        label="Pick a Gregorian date (clearable)"
+                        component={CalendarInput}
+                        calendar="gregory"
+                        locale="en-GB"
+                        clearable
+                />
+        </div>
+        <div>
+                <CalendarDemo
+                        label="Pick an Ethiopic date"
+                        date="2014-02-03"
+                        component={CalendarInput}
+                        calendar="ethiopic"
+                        locale="en-GB"
+                        clearable
+                />
+        </div>
+        <div>
+                <CalendarDemo
+                        label="Pick an Ethiopic date (not passing date)"
+                        component={CalendarInput}
+                        calendar="ethiopic"
+                        locale="en-GB"
+                        clearable
+                />
+        </div>
+        <div>
+                <CalendarDemo
+                        label="Pick an Ethiopic date (amharic)"
+                        component={CalendarInput}
+                        calendar="ethiopic"
+                        locale="am-ET"
+                        clearable
+                />
+        </div>
+        <div>
+                <CalendarDemo
+                        label="Pick a Nepali date (with date)"
+                        component={CalendarInput}
+                        calendar="nepali"
+                        locale="en-NP"
+                        date="2078-06-27"
+                        clearable
+                />
+        </div>
+        <div>
+                <CalendarDemo
+                        label="Pick a Nepali date (english)"
+                        component={CalendarInput}
+                        calendar="nepali"
+                        locale="en-NP"
+                        clearable
+                />
+        </div>
+        <div>
+                <CalendarDemo
+                        label="Pick a Nepali date (nepali)"
+                        component={CalendarInput}
+                        calendar="nepali"
+                        locale="ne-NP"
+                        clearable
+                />
+        </div>
+    </div>
+    <style jsx>{`
+        .wrapper {
+            display: flex;
+            flex-wrap: wrap;
+            gap: 10px;
+            overflow-x: scroll;
+            justify-content: space-around;
+        }
+        .wrapper > div {
+                /* min-width: 240px */
+        }
+    `}
+    </style>
+</Demo>
+
+## API Reference
+
+The component takes the same props as [the calendar](./calendar) component, as well as the props for [InputField](./inputfield).
```

Comment:
I guess a couple of these might not be relevant (because we're restricting to  type=text, and some of the props like `step` only apply when type=number)? I guess it's just a thought that we might clarify the documentation at some point?